### ISSUE TITLE
Detect reports should not show deleted endpoints

### ISSUE THREAD

**Issue by PeteC-Prelude** [preludeorg/build#685]
This is something to debate.  If I choose to delete a probe via CLI, it would seem appropriate to not show that probe in test results.  In the future, should consider a toggle for showing all data and showing only active probes.  

In this view, 4 probes that were deleted via CLI are still showing data in the console.
![screenshot_44](https://user-images.githubusercontent.com/126490894/221990274-037234ac-e24f-440d-9c62-338a75c779a3.png)

**Comment by mireaulf** [comment]
Those probes were deleted but their results are still there but since they don't bring back new results, you won't be seeing them as days pass

**Comment by PeteC-Prelude** [comment]
Understood.  My POV is that if I went through the effort of deleting it, I dont want to see it or have it cluttering my reports.  Aging is one method of keeping data clean, but deletion and also endpoint unique identifiers also come into play.

**Comment by kenrick** [comment]
Agreed, indicating the probe no longer exists is important along with having a away to hide deleted probes.

**Comment by privateducky** [comment]
@PeteC-Prelude please verify if the fix, just deployed, resolves your issue.

**Comment by PeteC-Prelude** [comment]
I see improvement in the situation, but some of the numbers are not lining up.  There are a number of variables at play, but the counts and percentages are not making sense.  
<img width="1280" alt="Screenshot 2023-02-28 at 6 57 12 PM" src="https://user-images.githubusercontent.com/126490894/222010415-15dac889-57bc-4473-8711-db27c2455a17.png">
<img width="1280" alt="Screenshot 2023-02-28 at 6 57 54 PM" src="https://user-images.githubusercontent.com/126490894/222010419-c45d6dda-b7a9-47c1-b2b6-68c82c39b3b3.png">

**Comment by mireaulf** [comment]
About the first image, the host count you are seeing is for the whole period selected. If you click on a bar on the graph you should see state for that day only.

**Comment by privateducky** [comment]
Tough to tell from a screenshot on this (for example, I can't tell if you've clicked on a line in the chart) but the only thing I see off is the top screen shot, where the 11/20 on the rules is not matching the 10/19 on the host grid. Given the "off by one" nature of that, my money is on a time zone difference in the calculation here and/or the fact this test is the our first example to date with multiple tests under 1 rule. 

@mireaulf @makk94 let me know what you guys find on this one.

**Comment by makk94** [comment]
I haven't been able to repro, even with two tests under 1 rule

![Screenshot 2023-02-28 at 6 22 32 PM](https://user-images.githubusercontent.com/9056078/222028758-97aa2576-68eb-4743-8853-306656691df7.png)

**Comment by kenrick** [comment]
I have an idea to address the rule `x hosts are failing`. As it appears windows filter is also applied in the second screenshot  which doesn't currently affect the count. I will ensure it does.

**Comment by PeteC-Prelude** [comment]
I can give access to this account if its helpful to diagnose?  I plan to add some additional probes today and dig into the numbers.
